regulations/static/regulations/js/source/views/comment/comment-confirm-view.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var $ = require('jquery');
+var URI = require('urijs');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var MainEvents = require('../../events/main-events');
+var PreambleHeadView = require('../header/preamble-head-view');
+var CommentEvents = require('../../events/comment-events');
+var comments = require('../../collections/comment-collection');
+
+var CommentConfirmView = Backbone.View.extend({
+  initialize: function(options) {
+    Backbone.View.prototype.setElement.call(this, '#' + options.id);
+
+    this.findElms();
+
+    this.docId = this.$el.data('doc-id');
+    this.metaUrl = this.$el.data('meta-url');
+
+    this.listenTo(CommentEvents, 'read:proposal', this.handleRead);
+
+    if (this.metaUrl) {
+      this.poll(this.metaUrl);
+      comments.filter(this.docId).forEach(function(comment) {
+        comment.destroy();
+      });
```

Comment:
Prefer a unit test for the view or fleshing out the existing end-to-end comment submit test? Or both?